### ISSUE TITLE
[问题]有没有完全卸载的方法。

### ISSUE THREAD

**Issue by love768952693** [wulabing/Xray_onekey#349]
在非全新安装的机器上，每次都卡在最后一步。[ERROR]  Nginx config modify 失败 
多次安装，提示成功，发现网站打不开，默认443端口也没监听。。。直接通过IP访问能看到Nginx欢迎页。

所以我想卸载干净再安装，用过博主的命令也没用，手动删除过，V2Ray，Nginx，不知道为什么。。急哭了。。。
`wget -N
https://raw.githubusercontent.com/wulabing/V2Ray_ws-tls_bash_onekey/master/install.sh
&& bash ./install.sh uninstall`

**Comment by KukiSa** [comment]
@love768952693 In order to accurately locate the problem, please provide the following information:
 1. Before executing the script, what software related to nginx or v2ray did you install?
 2. Please execute ``which nginx`` and paste the output.

**Comment by love768952693** [comment]
之前用过你们的，另外一个脚本 V2Ray的，目前我已经重装系统了，我觉得这样的方式更加轻松。。。。。。。。。谢谢了。

**Comment by KukiSa** [comment]
@love768952693 Yes, reinstalling system is the most thorough and safest solution.
The problems you encountered, as you analyzed, are precisely caused by the improper location of the old environment (especially nginx).  In addition, there are deeper problems such as "V2ray installation location conflict", "nginx configuration file tree exception" and so on. The operations required to solve these problems are very complicated.
Congratulations on making the right choice and wish you a happy using!
